kevlar/tests/test_novel.py

```diff
@@ -58,6 +58,7 @@ def test_assumptions(kmer):
     assert ct.get_kmer_hashes(kmer)[0] == ct.get_kmer_hashes(kmer_rc)[0]
 
 
+@pytest.mark.long
```

Comment:
Uses deprecated code anyway, so no sense in taking the time to run it locally with each `make test` invocation.